### DISCUSSION TITLE
How can I disable `nixpkgs-nixvim-doc` attribute ?

### DISCUSSION THREAD

**Discussion by 573** [nix-community/nixvim#1296] [Q&A] [answered]
I'm using nixvim with custom configuration via flake and when building the flake `nixpkgs-nixvim-doc` is built i. e.:

>2024-03-20T10:25:05.7687188Z copying path '/nix/store/fckxq70575d0rfd884kriwz9wzg39jfq-nixvim-configuration-reference-manpage' from 'https://nix-community.cachix.org'...
2024-03-20T10:24:25.6417449Z building '/nix/store/b4p4jzyg6yv23nk40szhg003y5pm3g93-nixpkgs-nixvim-doc.drv'...

**Comment by 573** [comment]
I think I know now why `nixpkgs-nixvim-doc` is built, using it in my flake like `inputs.nixvim.legacyPackages."${system}".makeNixvim { colorschemes.gruvbox.enable = true; };` (thus https://github.com/nix-community/nixvim/blob/5d3ed3a/flake-modules/packages.nix#L9).
The https://github.com/nix-community/nixvim/tree/main/templates should work better.

**Comment by traxys** [comment, marked as answer]
You would be able to disable the man page (using `enableMan = false`) to stop building the man page. The nixpkgs-nixvim-doc is required to build the man page.

**Comment by 573** [comment]
https://github.com/nix-community/nixvim/blob/d6a6685/wrappers/hm.nix#L54
